doc/install.text

```diff
@@ -0,0 +1,124 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
+* Install git-lfs (https://git-lfs.github.com/)
+* Install matplotlib:
+
+conda install matplotlib
+
+
+If you have a very fresh compiler version, this could lead to incompatibilities 
+with the precompiled LSST stack. In case you run into trouble compiling or testing
+the LSST stack please install your compiler in anaconda:
+
+conda install gcc
+
+
+* Create a directory for the PFS-DRP repositories:
+
+mkdir ~/stella-git
+cd ~/stella-git
+
+* Clone the relevant git repositories:
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+
+
+* Install a newer version of eigen:
+
+eups distrib install eigen 3.2.5.lsst2
+
+
+* Declare the git repositories in Linux (replace <Linux64> with <DarwinX86> for Mac OS X):
+
+eups declare obs_pfs 1.0 -c -f Linux64 -r ~/stella-git/obs_pfs
+eups declare drp_stella_data 1.0 -c -f Linux64 -r ~/stella-git/drp_stella_data
+eups declare drp_stella 1.0 -c -f Linux64 -r ~/stella-git/drp_stella
```

Comment:
Do you need to specify the architecture? Seems to work fine without for me.

Do you want to declare these as version 1.0? That sounds like a finished product. I don't think there is a usable versioning scheme, but you could make something up from the date (`echo 0.0.$(date +%Y%m%d)`) or following PEP-440 or something. Or just make them 0.1 instead!